crux_kv/src/lib.rs

```diff
@@ -205,7 +205,34 @@ where
             };
         }
 
-        response
+        response.into()
+    }
+}
+
+impl From<KeyValueResult> for Result<bool, KeyValueError> {
```

Comment:
Personally I don't like `From` impls which can panic or do anything more than the 'obvious' thing. You don't get a lot of benefit from using the trait, and it ends up with an unhelpful method name. `my_kv.into()` isn't as good as eg `my_kv.unwrap_exists()` or `my_kv.unwrap_value`, something which indicates you are asserting some truth about the variable.

Reply:
Would `TryFrom` impl be better? We could map the error

Reply:
Or even map the error in the `From` impl, actually

Reply:
Yea, perhaps a straight up impl methods that convert - maybe something like `expect_[operation]` - would be better

Reply:
`unwrap_` is a better name, just for the record.